src/fr/pages/index.md

```diff
@@ -72,8 +72,8 @@ date: "git Last Modified"
 
 <article class="py-450">
   <h2 class="mb-400">Composant vedette</h2>
-  <h3 class="mb-400">Jeu de champs</h3>
-  <p class="mb-450">Le jeu de champs permet aux utilisateur·rice·s de percevoir un groupe d'éléments de formulaire connexes. Les éléments ainsi groupés, soit de façon logique et accessible, sont plus faciles à utiliser et comprendre.</p>
-  <img class="d-block mb-450" src="../../images/common/components/preview-fieldset.svg" alt="UI preview of fieldset component using an input and a select field." alt="Un aperçu du composant jeu de champ qui montre une suite de plusieurs éléments. Une boîte grise foncée représentant le titre du groupe suivi d'une boîte grise représentant du texte. On voit ensuite une boîte de texte vide avec le curseur à l'intérieur suivi d'une boîte grise représentatn du texte et une boîte de sélection avec une boîte grise à l'intérieur représentant du texte." />
-  <a href="{{ links.fieldset }}">En savoir plus sur le jeu de champs</a>
+  <h3 class="mb-400">Barre de navigation supérieure</h3>
+  <p class="mb-450">Une barre de navigation supérieure est une liste horizontale de liens de page. Utilisez une barre de navigation supérieure pour aider une personne à se repérer sur votre page Web ou site Web.</p>
+  <img class="d-block mb-450" src="../../images/common/components/preview-top-nav.svg" alt="UUn aperçu du composant barre de navigation supérieure qui montre la navigation du site représentée par des boîtes grises alignés horizontalement. Une boîte bleue suivi de deux boîtes grises représentent les liens où la dernière boîte est surlignée afin de représenter le lien actif." />
```

Comment:
Small typo - extra "U" at the start of the alt text

Reply:
Good catch!